docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  database:
+    container_name: postgres-betest
+    image: postgres:16.3
+    restart: always
+    ports:
+      - ${DATABASE_PORT}
+    env_file:
+      - ".env"
+    environment:
+        - POSTGRES_USER=${DATABASE_USER}
+        - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
+        - POSTGRES_DB=${DATABASE_NAME}
+
+networks:
+  fullstack-project:
+    driver: bridge
+
+volumes:
+  postgres-data:
```

props for defining networks and volumes 👍🏻 But you are not using them with the database container - you can do it this way:
```suggestion
services:
  database:
    container_name: postgres-betest
    image: postgres:16.3
    restart: always
    ports:
      - ${DATABASE_PORT}
    env_file:
      - ".env"
    environment:
        - POSTGRES_USER=${DATABASE_USER}
        - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
        - POSTGRES_DB=${DATABASE_NAME}
    volumes:
      - postgres-data:/var/lib/pgsql/data
    networks:
      - fullstack-project

networks:
  fullstack-project:
    driver: bridge

volumes:
  postgres-data:
```

Note that `/var/lib/pgsql/data` path is to be verified. Alternatively you can add:

```
    environment:
      PGDATA: /data/postgres
```
to service and then modify `volumes`:
```
    volumes:
      - postgres-data:/data/postgres
```